src/main/java/org/reaktivity/specification/nukleus/oauth/internal/OAuthFunctions.java

```diff
@@ -68,6 +77,23 @@ public static JwtHelper jwt(
         return helperFactory.get();
     }
 
+    static JwtHelper jwt(
+        KeyPair pair,
+        String kind,
+        String algorithm)
+    {
+        Map<String, Supplier<JwtHelper>> helperFactories = new HashMap<>();
+        helperFactories.put(kind, () -> new JwtHelper(pair, kind, algorithm));
+        Supplier<JwtHelper> helperFactory = helperFactories.get(kind);
+        return helperFactory.get();
```

Comment:
No need to populate a `helperFactories` map, just need to `return new JwtHelper(pair, kind, algorithm)` directly.